pip/tests-integration/interop_qiskit/test_circuits/random.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import pytest
+from qsharp import TargetProfile
+
+from .. import QISKIT_AVAILABLE
+
+if QISKIT_AVAILABLE:
+    from qiskit.circuit.random import random_circuit
```

Comment:
Is this really what it sounds like? Generates a random circuit? I'm confused by why this would exist in Qiskit.

More relevant to this PR, I don't think it's a great idea to have nondeterministic tests in the build. It seems that you're already explicitly covering edge cases pretty well?

Reply:
They had value when they were written. Less so now, removed them.